bpf/lib/nodeport.h

```diff
@@ -576,61 +575,60 @@ static __always_inline int nodeport_lb6(struct __ctx_buff *ctx,
 		return DROP_MISSED_TAIL_CALL;
 	}
 
-	ret = ct_lookup6(get_ct_map6(&tuple), &tuple, ctx, l4_off, CT_EGRESS,
-			 &ct_state, &monitor);
-	if (ret < 0)
-		return ret;
 	if (!revalidate_data(ctx, &data, &data_end, &ip6))
 		return DROP_INVALID;
 
 	backend_local = lookup_ip6_endpoint(ip6);
 	if (!backend_local && lb6_svc_is_hostport(svc))
 		return DROP_INVALID;
 
-	switch (ret) {
-	case CT_NEW:
+	if (backend_local || (!backend_local && !nodeport_uses_dsr6(&tuple))) {
```

Comment:
```suggestion
	if (backend_local || !nodeport_uses_dsr6(&tuple)) {
```
Maybe also worth adding a small comment with basically what's in the commit message?